// Protocol buffer for the cache configuration
// By: Nick from CoffeeBeforeArch
// 4/15/2020

// Configuration settings for each CacheLevel
message CacheConfig {
  // Size of the cache (in bytes)
  required int64 cache_size = 1;

  // Size of a cache line (in bytes)
  required int32 line_size = 2;

  // Associativity
  required int32 associativity = 3;
}
